### ISSUE TITLE
Setting rewrite rules for custom post type

### ISSUE THREAD

**Issue by binhwpo** [xydac/WP-Ultimate-CMS#8]
Hi @xydac,

I'm trying to add rewrite rule to custome post type using %taxomony_name%/type in the **Slug** but it doesn't work at all.

Any hint?

**Comment by xydac** [comment]
I remember there was an issue, but don't remember what :(
You might want to look at `modules/post-type/class-xydac-post-type.php`

May be check what is being set in `$xy_cpt['args']['rewrite'] `
just before line 86 in above mentioned file.
-Deepak

**Comment by binhwpo** [comment]
@xydac 

I see nothing wrong with it. The problem here is that I want to add permalink like this

/%custom_tax%/%post_type%/post-slug

Example for a location specific:

/india/hotels/some-hotel-name

I saw the line `$wp_rewrite->add_rewrite_tag('%'.$this->query_var.'%', '([^/]+)', $this->query_var . '=');`
in `class-custom-post-type-with-rewrite-rules.php` but I see no where including the file `class-custom-post-type-with-rewrite-rules.php`

**Comment by binhwpo** [comment]
<<NOT USING THIS NOW>>

**Comment by xydac** [comment]
Yes i started work on that but never merged. That file is not included anywhere. That's why i gave you reference to file you needed to check the values at ?

Thanks,
Deepak

**Comment by binhwpo** [comment]
@xydac I'm improving a plugin on the custom permalink. If it's suitable I will try to merge it into this.
